RULEAPI-706: Add quick fixes metadata (#459)
SonarSource/rspec: README.adoc

.. Bob merges the rule implementation in Cobol analyzer.
.. If Bob had opened an RSPEC PR with changes, he merges it once the rule is implemented in the analyzer.

== Quickfix field

Every active rule that is not a security hotspot must specify the availability of a quickfix for its issues.
`metadata.json` must feature a `quickfix` field with one of the following values:
* `unknown`: the feasibility of producing a quickfix is not evaluated.

> * `infeasible`: it is not feasible to propose a quickfix for any of the issues produced by the rule.

Reviewer: I feel that we could implicitly mention that it can be either due to technical limitations or it is simply impossible. For example: "it is not feasible to propose a quickfix for any of the issues, for whatever reason."

> * `targeted`: it is possible to implement automatic quickfixes at least for some of the issues produced by the rule, but none are implemented right now.

Reviewer: ```suggestion
* `targeted`: it is possible to implement quickfixes for this rule, but none are implemented right now.
```
